.eslintrc.js

```diff
@@ -101,6 +101,16 @@ module.exports = {
     'simple-import-sort/imports': 'warn',
 
     'import/newline-after-import': ['warn', { 'count': 2 }],
-    'import/no-anonymous-default-export': ['warn', { 'allowObject': true }]
-  }
+    'import/no-anonymous-default-export': ['warn', { 'allowObject': true }],
+
+    'no-debugger': 'warn'
+  },
+  'overrides': [
+    {
+      'files': 'src/**/*.js',
+      'rules': {
+        'no-console': ['warn', { 'allow': ['assert', 'error'] }]
```

Comment:
There were enough `console.assert` and `console.error`s that I left them enabled instead of going through and adding `eslint-disable-line` comments to each one.

Reply:
I think console.assert is fine in code.